### DISCUSSION TITLE
Does this work on Embedded Systems, like Orange PC PI H3 or Orange PI 3 H6?

### DISCUSSION THREAD

**Discussion by CommodoreOS** [oomek/attractplus#15] [Q&A]
So can I use the armv7 version on the Alwinner H3 and the AArch64 version on the H6...?

Is there any other stuff I need to get this to work, dependencies and that, do I need original Attract Mode so its files that are in the attract folder in the RPI3 version.

I'm the creator of PCUAE/PCUAAE - Project Carousel USB: Anniversary Edition - Project Carousel USB: Amiga Anniversary Edition, PCUAE runs on THEC64 and PCUAAE runs on THE A500 Mini... https://github.com/CommodoreOS/PCUAE

I am trying to get Attract Mode working on THEA500 Mini( its a mini console, it was made by Retro Games LTD and released in April, made by the people behind THEC64, its like the SNES Mini, but for Amiga, it looks like A500 but at 50% scale, it here... https://retrogames.biz/products/thea500-mini ), and in PCUAAE, its specs is the same as the Orange PI 3 H6 but only has 512mb ram not 1 GB, it uses a 3.10 kernel to boot Buildroot from its NAND and then runs its carousel, it uses Amiberry to run Amiga Apps or Games , it can load ADF or  HDF files, Buildroot is 64bit cause its needed to get Amiberry v3.3 working on it, it uses a comised version of SDL2 on it so Amiberry works with the SoC H6.

I am trying to get it to work on it cause it carousel is locked down so it would need hacking a lot to add games to it so would be easier to add AM to it, and you can make Attract Mode look like THEA500 Mini's carousel so make a theme that looks like it, the carousel is a copy of the Amazon Prime Video GUI so I want to replace its carousel with AM.
It can run Retroarch 1.10.3(so could link Attract Mode to Retroarch so it loads the games from RA), the RA mod is called Pandory... https://github.com/emuchicken/pandory-a500 - Pandory and PCUAAE on THEA500 Mini - https://www.youtube.com/watch?v=0vIu6DXyKhA

Any help would be appreciated.. thanks.. :)

**Comment by concentrateddon** [comment]
AM+ will only run on an operating system for which it's been compiled. There are numerous releases for different OSs, but you'd need to either be running one of those, or entirely recompile the code to run on a different OS.
